Using-browsec-for-securing-your-connection-to-fcc.md

```diff
@@ -0,0 +1,63 @@
+# Using Browsec for Securing your Connection to FreeCodeCamp.com
+
+## Why do I need an plugin?
+Lately some of the campers started facing a strange problem, namely content blocking by some Internet Service Providers (ISP), leading to the [FreeCodeCamp.com](http://freecodecamp.com) (FCC) website being broken sometimes.
```

Comment:
Grammar:
> facing strange problems.

In general we're not so much seeing "content blocking" as "content modification".

but the real "problem" is not so much that the content is being modified as that the modification is breaking the site.  It's a fine distinction, but important.